operators/hack/run-e2e.sh

```diff
@@ -81,6 +81,14 @@ else
     echo "--"
     kubectl -n $NAMESPACE get elastic,pods,rs,deploy,svc,cm,secrets,pvc,pv
     echo "--"
+    echo "# events for non-running pods"
+    for pod in $(kubectl get pods -n $NAMESPACE | tail -n +2 | grep -v Running | awk '{print $1}')
```

Comment:
FWIW, a more kubectl-ish version could be:

```
kubectl -n $NAMESPACE get pods \
	--no-headers \
	--field-selector=status.phase=Running \
	-o custom-columns=:metadata.name \
| xargs -I {} kubectl -n $NAMESPACE get event \
	--field-selector involvedObject.name={}
```

Reply:
Indeed that looks better

Reply:
I'll remove the xargs part which makes it harder to chain "echo $pod && cmd", keep the for loop but use your kubectl args.